dags/vm_resource.py

```diff
@@ -98,11 +99,14 @@ class ClusterName(enum.Enum):
   V4_32_CLUSTER = "mas-v4-32"
   V5E_4_CLUSTER = "mas-v5e-4"
   V5E_16_CLUSTER = "mas-v5e-16"
+  V4_8_MULTISLICE_CLUSTER = "v4-8-maxtext"
```

Comment:
Could you also help add `v4-16-maxtext` for future cases?

Reply:
Done!